README.md

```diff
@@ -14,17 +14,42 @@ The project is built on a plugin model, which ensures customizability and wide a
 
 ## Running the app
 
+### Development
+
 ```bash
 # Copy env file
 $ cp .env.copy .env
+
+# Specify the local compose file
+$ docker-compose -f docker-compose.local.yml up -d
+
+# Installing app dependencies
+$ yarn install
+
+# Run Prisma migrations
+$ yarn prisma migrate dev -n init
+
+# Start in watch mode
+$ yarn start:dev
 ```
 
+### Production
+
 ```bash
-# Specify the local compose file
-$ docker-compose -f docker-compose.local.yml up -d --build
+# Copy env file
```

Comment:
add a comment something like "You need to edit .env according to your prod setup" ...